addons/mrp_subcontracting_dropshipping/models/res_company.py

```diff
@@ -40,7 +77,32 @@ def create_missing_subcontracting_dropshipping_rules(self):
         company_todo_rules = company_ids - company_has_rules
         company_todo_rules._create_subcontracting_dropshipping_rules()
 
+    @api.model
+    def create_missing_subcontracting_dropshipping_sequence(self):
```

Comment:
Make it private.
```suggestion
    def _create_missing_subcontracting_dropshipping_sequence(self):
```

Reply:
its used in the migration script and xml functions, are you sure it should be private?